lldb/source/Plugins/DynamicLoader/POSIX-DYLD/DynamicLoaderPOSIXDYLD.cpp

```diff
@@ -572,9 +572,14 @@ ModuleSP DynamicLoaderPOSIXDYLD::LoadInterpreterModule() {
   ModuleSpec module_spec(file, target.GetArchitecture());
 
   if (ModuleSP module_sp = target.GetOrCreateModule(module_spec,
-                                                    true /* notify */)) {
+                                                    false /* notify */)) {
```

Comment:
Might as well change to the [official style](https://llvm.org/docs/CodingStandards.html#comment-formatting): `/*notify=*/false`

Reply:
Can you put a comment in here saying why you pass false here.  The next person who comes to read this won't have the context of this discussion and will likely be confused.

Reply:
Updated coding style and added a comment here.